frontend-ssr/redux/root.reducer.ts

```diff
@@ -1,8 +1,8 @@
 import { combineReducers } from 'redux';
-import { dummy } from './dummy/dummy-reducer';
+import { submissionReducer } from './exercise-submissions/exercise-submissions.reducer';
 
 const rootReducer = combineReducers({
-  dummy,
+  submissionReducer,
```

Comment:
The state is a plain object so the property names should not be related to redux.

So here, let's change to

```
import {submissionReducer as submissions} from '...';

combineReducers({
  submissions
})
```